### PR TITLE
Add collateral screen + refactor/remove old code

### PR DESCRIPTION
**Pull request by nistadev**
Added new screen for adding collateral tx.

Task: [YOEXT-395](https://emurgo.atlassian.net/browse/YOEXT-395)

**Sreenshot**
image

### CODE REVIEW

**Review by zuzunker — APPROVED** [review]
Nice! Thank you, @nistadev !

**Review by zuzunker — COMMENTED** [review]
/check

File: packages/yoroi-extension/app/ergo-connector/components/signin/AddCollateralPage.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    defaultMessage: '!!!Incorrect wallet password.',
+  },
+  reorgTitle: {
+    id: 'connector.signin.reorg.title',
+    defaultMessage: '!!!Add Collateral',
+  },
+  reorgMessage: {
+    id: 'connector.signin.reorg.message',
+    defaultMessage:
+      '!!!<span>To interact with <a href="https://emurgohelpdesk.zendesk.com/hc/en-us/articles/4415793858959-What-are-smart-contracts" target="_blank">smart contract</a> in Cardano you should add collateral, which means to make a 0 ADA transaction.<br/><br/>It is a guarantee that prevent from failing smart contracts and scams. <a href="https://docs.cardano.org/plutus/collateral-mechanism" target="_blank">Learn more</a> about collateral.</span>',
```
Comment (neuodev) [inline packages/yoroi-extension/app/ergo-connector/components/signin/AddCollateralPage.js]: Notice that the `a` tag and the `href` will not be translated! take a look [here](https://github.com/Emurgo/yoroi-frontend/blob/bcc07d8536e85a4c61a70f9282538d69a479f1a7/packages/yoroi-extension/app/components/loading/Crashed.js#L63,L79) and see how you can remove them and at the same time integrate it with the rest of the text. Also this `href` might be updated at any time! We should remove it from the translation so we don't have to update it for all languages later on.

Comment (nistadev) [reply packages/yoroi-extension/app/ergo-connector/components/signin/AddCollateralPage.js]: Great! Thank you, pushing an update.

**Comment by neuodev** [comment]
The page looks good I suggest adding a `max-width` so it looks good on large screens 

<details>
<summary>Current</summary>

Screen Shot 2022-10-06 at 1 08 52 PM

</details>

<details>
<summary>Example</summary>

Screen Shot 2022-10-06 at 1 10 56 PM

</details>

**Comment by nistadev** [comment]
> The page looks good I suggest adding a `max-width` so it looks good on large screens
> 
> Current
> Example

Fixed it! Thanks! :point_down: 
<details>
<summary>Wide Screen</summary>
Screen Shot 2022-10-06 at 1 10 56 PM
</details>

File: packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js [hunk 2]
```diff
@@ -206,18 +197,17 @@ class SignTxPage extends Component<Props, State> {
...
-    const shiftedAmount = request.entry.amount.shiftedBy(- numberOfDecimals);
-    const ticker = tokenInfo ? this.getTicker(tokenInfo)
+    const shiftedAmount = request.entry.amount.shiftedBy(-numberOfDecimals);
+    const ticker = tokenInfo
+      ? this.getTicker(tokenInfo)
       : assetNameFromIdentifier(request.entry.identifier);
 
     let fiatAmountDisplay = null;
 
+    // this is a feature flag
```
Comment (neuodev) [inline packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js:207]: When I enable it. I found itlooks like this!
Screen Shot 2022-10-06 at 1 32 02 PM

**Review by neuodev — CHANGES_REQUESTED** [review]

File: packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js (hunk 2 above)
Comment (nistadev) [reply packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js:207]: Not on the scope of this task, just old code that is there (you can see the forced `false`). This file was formatted, linted and I removed the old code that was used to show the collateral message, not done anything else :smile:

Comment (neuodev) [reply packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js:207]: Got it! I though you should be updating this as well

Comment (neuodev) [reply packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js:207]: <details>
<summary> As I might forget to fix it later. I fixed here 😅 </summary>
Screen Shot 2022-10-06 at 1 58 42 PM

</details>

**Review by neuodev — APPROVED** [review]
Sounds good! Thanks @nistadev 👍

**Review by zuzunker — COMMENTED** [review]
/check

**Pull request merged**